exif-reader.d.ts

```diff
@@ -37,6 +37,7 @@ interface ExpandedTags {
     exif?: Tags,
     iptc?: Tags,
     xmp?: XmpTags
+    icc?:any // TODO: add
```

Comment:
Reminder to not forget this.

Reply:
Please take a look at my changes, not really fluent with typescript.